service/src/main/java/uk/gov/hmcts/reform/fpl/service/MigrateCaseService.java

```diff
@@ -57,6 +58,72 @@ public class MigrateCaseService {
     private final CourtService courtService;
     private final DocumentListService documentListService;
 
+    public Map<String, Object> rollbackCourtBundleMigration(CaseData caseData) {
+        List<Element<HearingCourtBundle>> newHearingCourtBundleList = new ArrayList<>();
+
+        newHearingCourtBundleList.addAll(caseData.getHearingDocuments().getCourtBundleListV2());
+        newHearingCourtBundleList.addAll(caseData.getHearingDocuments().getCourtBundleListLA());
+        newHearingCourtBundleList.addAll(caseData.getHearingDocuments().getCourtBundleListCTSC());
+
+        Map<String, Object> ret = new HashMap<>();
+        ret.put("courtBundleListV2", newHearingCourtBundleList);
+        ret.put("courtBundleListLA", emptyList());
+        ret.put("courtBundleListCTSC", emptyList());
```

Comment:
should pass the `CaseDetails` to this method and do the following:
`caseDetails.getData()..remove("courtBundleListLA")`
`caseDetails.getData()..remove("courtBundleListCTSC")`